saleor/dashboard/group/views.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import unicode_literals
+
+from django.shortcuts import redirect, get_object_or_404
+from django.contrib import messages
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from django.utils.translation import pgettext_lazy
+
+from .forms import GroupPermissionsForm
+from ..views import staff_member_required
+
+
+@staff_member_required
+def groups_list(request):
+    groups = Group.objects.all()
+
+    ctx = {'groups': groups}
+    return TemplateResponse(request, 'dashboard/group/list.html', ctx)
+
+
+@staff_member_required
+def group_create(request):
+    form = GroupPermissionsForm(request.POST or None)
+
+    if form.is_valid():
+        form.save()
+        return redirect('/dashboard/groups')
```

Comment:
You should always use `reverse` to get the URL. Also a success message would be nice before redirecting to the list view. We use this pattern in all create/edit/delete views in dashboard.

Reply:
Done